packages/redwood/src/index.ts

```diff
@@ -1,4 +1,5 @@
 import { join, dirname, relative, parse as parsePath, sep } from 'path';
+import { statSync } from 'fs';
```

Comment:
Can be removed with my suggestion [below](https://github.com/vercel/vercel/pull/4999/files#r466688471).

```suggestion
```